### DISCUSSION TITLE
How do I add Layouts like Rich yet in Textual and How can the data / information inside of a placeholder be changed ?

### DISCUSSION THREAD

**Discussion by regional-specter** [Textualize/rich#3051] [Q&A] [answered]
I'd like to start by saying I am a beginner to `Textual` yet have had a lot of experience using rich, and was wondering how could i replicate that layout function that we all use simply in textual using placeholders

**Comment by willmcgugan** [comment, marked as answer]
The short answer is "yes". It works a little different in Textual but you can create almost any layout. This link may get you started:

https://textual.textualize.io/how-to/design-a-layout/

**Reply by regional-specter** [reply]
Thanks so much Mr. Will, or may I say a true coding legend
